mage_ai/data_preparation/templates/transformers/data_warehouse_transformer.jinja

```diff
@@ -0,0 +1,27 @@
+from mage_ai.data_preparation.repo_manager import get_repo_path
+from mage_ai.io.io_config import IOConfig
+from mage_ai.io.{{ data_source }} import {{ data_source_handler }}
+from pandas import DataFrame
+from os import path
+
+if 'transformer' not in globals():
+    from mage_ai.data_preparation.decorators import transformer
+
+
+@transformer
+def transform_in_{{ data_source }}(*args) -> DataFrame:
+    """
+    Performs a transformation in {{ data_source_handler }}
+    """
+    config_path = path.join(get_repo_path(), 'io_config.yaml')
+    config_profile = 'default'
+
+    # Specify table to sample data from. Use to visualize changes to table.
+    sample_table = 'table_to_sample_data_from'
+    sample_schema = 'schema_of_table_to_sample'
+    sample_size = 10_000
+
+    with {{ data_source_handler }}.with_config(IOConfig(config_path).use(config_profile)) as loader:
+        # Write queries to transform your dataset with
+        loader.query('your transformation_query'){{ additional_args }}
```

Comment:
Let's create a variable for the sql query so that user has more space to write sql instead of putting it in one line.

Reply:
Added variable for query